js/src/product-feed/review-request/review-request-issues.js

```diff
@@ -32,7 +33,7 @@ const ReviewRequestIssues = ( { issues = [] } ) => {
 			</Text>
 			<ul className="gla-review-request-modal__issue-list">
 				{ issuesToRender.map( ( issue ) => (
-					<li key={ issue }>{ issue }</li>
+					<li key={ issue.code }>{ issue.issue }</li>
```

Comment:
❔

I suppose that each `issue.code` is unique and can not be repeated otherwise we will have the warning: Encountered two children with the same key

Reply:
Yes, I filtered by Unique [in the backend Part PR](https://github.com/woocommerce/google-listings-and-ads/pull/1413)

Thanks for confirming